quipucords/quipucords/featureflag.py

```diff
@@ -0,0 +1,49 @@
+"""Filters feature flags from systen env variables."""
+
+import logging
+import os
+
+DEFAULT_FEATURE_FLAGS_VALUES = {
+    'OVERALL_STATUS': False}  # default values for feature flags that
+# are already implemented
+
+VALID_VALUES_FOR_ENV_VARIABLES = ['0', '1']
+
+# Get an instance of a logger
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def get_feature_flags_from_env():
+    """Filter feature flags from environment variables."""
+    global DEFAULT_FEATURE_FLAGS_VALUES
+    global VALID_VALUES_FOR_ENV_VARIABLES
+    for key, value in os.environ.items():
+        if key.upper().startswith('QPC_FEATURE_'):
+            feature = key.upper().replace('QPC_FEATURE_', '')
+            try:
+                if value in VALID_VALUES_FOR_ENV_VARIABLES:
+                    DEFAULT_FEATURE_FLAGS_VALUES[feature] = bool(int(value))
+            except ValueError:
+                logger.info(' %s can`t be converted to int, verify your '
+                            'environment variables.', value)
+                continue
```

Comment:
Another thing that comes from this if: invalid values are silently ignored.

As an user I'd rather know I did something wrong (either with an warning or just throwing an error preventing the application to start). 

(I guess we forgot to discuss what should be the proper behaviour of this class when invalid values are passed).

Here's my suggestion:

```suggestion
            feature_flags = DEFAULT_FEATURE_FLAGS_VALUES.copy()
            try:
                feature_flags[feature] = bool(int(value))
            except ValueError:
                raise ValueError(f'{value} is invalid for feature '{feature}'. Verify your environment variables.')
```